tests/test_environments/test_registry_override.py

```
@@ -152,6 +152,52 @@ def test_make_harbor_dispatch_pre_staged(tmp_path, monkeypatch, uri, expected_ca
     assert env.name == expected_cache_subdir
 
 
+@pytest.mark.parametrize(
+    "params, expected_keep_hints",
```

Ruff (0.15.17)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)